addons/dark-www/experimental_scratchr2.css

```diff
@@ -1,170 +1,660 @@
+.isticky {
+  filter: var(--darkWww-link-scratchr2IconFilter);
+}
+
+/* Footer background */
 #footer {
-  background-color: #333;
-  border-color: #606060; /* for infinite scrolling fixed footer */
-  box-shadow: none;
+  background-color: var(--darkWww-footer-scratchr2);
 }
 #footer ul.footer-col li h4 {
-  color: #fff !important; /* !important is needed, see #701 */
+  color: var(--darkWww-footer-scratchr2Text) !important; /* !important is needed, see #701 */
 }
-#topnav {
-  box-shadow: none;
+
+/* Border color */
+.modal-header,
+#profile-box .player,
+.nvtooltip h3 /* statistics */ {
+  border-color: var(--darkWww-border-5);
 }
-#topnav form.search .glass {
-  border-right: none;
-  background: url("https://scratch.mit.edu/images/nav-search-glass.png") no-repeat center center;
-  background-size: 14px 14px;
+.box .box-content .action-bar .inner {
+  border-bottom-color: var(--darkWww-border-5);
 }
-#topnav form.search input[type="text"] {
-  background: rgba(0, 0, 0, 0.1);
+.modal-footer,
+#profile-box .doing,
+#profile-data .footer,
+#email-resend-box {
+  border-color: var(--darkWww-border);
+}
+.nvd3 .nv-axis line /* statistics */ {
+  stroke: var(--darkWww-border);
+}
+#footer /* for infinite scrolling fixed footer */,
+.thumb img,
+.djangobb blockquote,
+.my-ocular-popup,
+.media-list li,
+.media-item-content .media-thumb img,
+.header-text .profile-details .group,
+.slider-carousel .thumb img,
+#comments #comments-enabled-box,
+.table td,
+#comments .sa-emoji-picker,
+.sa-emoji-picker-divider {
+  border-color: var(--darkWww-border-15);
+}
+.markItUpHeader ul ul li:hover {
+  border-color: var(--darkWww-border-15) !important;
+}
+.my-ocular-popup::before {
+  border-top-color: var(--darkWww-border-15);
+}
+.v-tabs li:hover {
+  border-bottom-color: var(--darkWww-border-15);
 }
-#topnav form.search input[type="text"]::placeholder {
-  color: rgba(255, 255, 255, 0.75);
+input,
+textarea,
+select,
+.jqui-modal,
+.jqui-modal.ui-corner-all,
+#topnav form.search .glass,
+#page-404 .search input,
+.box .box-head .portrait img,
+.gallery.thumb span.image,
+.gallery.thumb span.image img,
+#comments #comment-form .template-feature-off,
+#comments .comments li,
+#comments ul.comments ul li,
+#comments .more-replies,
+#comments .more-replies .pulldown,
+.nvtooltip /* statistics */,
+.help .faq .box-content > div,
+.registration a.why-we-ask:hover+div {
+  border-color: var(--darkWww-border-20);
+}
+.editable:hover,
+.editable.write,
+.editable-empty {
+  border-color: var(--darkWww-border-20) !important;
 }
-#topnav ul.account-nav .logged-in-user .dropdown-menu {
-  border: 1px solid rgba(0, 0, 0, 0.1);
+.markItUpPreviewFrame {
+  border-color: var(--darkWww-border-20) !important;
+  background-color: transparent !important;
+}
+.dropdown .dropdown-menu {
   background-clip: border-box;
+  border-color: var(--darkWww-border-20);
+  border-top-color: var(--darkWww-border-5);
+}
+.djangobb hr {
+  background-color: var(--darkWww-border-20);
+}
```

Comment:
@mxmou
From feedback:
> The signature line uses the border colour, I have borders hidden to make the UI cleaner
i wanna remove the other borders but not that one
i still wanna hide the borders for a cleaner ui while showing the signature line for clarity

Reply:
Personally I believe the request is reasonable, I wouldn't consider the signature line to be a "border". And I believe making borders 100% transparent might be pretty common.
But I don't have any concrete suggestions on how else we could compute an appropriate color for the signature line. Of course, we wouldn't want to add a specific setting for it.

Reply:
One option is to generate a different color for the signature line if the border color is fully transparent.

Reply:
@mxmou Sounds good to me. Should I create an issue?

Reply:
What about making it 50% opaque text colour?

Reply:
@Secret-chest Keep in mind that the color would need to stay the same under "Scratch's default colors" preset